### ISSUE TITLE
1.5.x: ucx_info -b, ENABLE_MT mismatch

### ISSUE THREAD

**Issue by tonycurtis** [openucx/ucx#3743]
Seen with 1.5.1 and 1.5.2.  Git version works.

```shell
$ .../configure ... --enable-mt ...
$ grep _MT config.h
#define ENABLE_MT 1
```

Install, then

```shell
$ ucx_info -b | grep _MT
#define ENABLE_MT                 0
```

FWIW `ENABLE_PARAMS_CHECK` is also mismatched.

**Comment by shamisp** [comment]
Config logic has been busted for a while now. I think #3715 fixes this. It might be too late for 1.5.x :-/

**Comment by tonycurtis** [comment]
Obviously didn't find the right search string to see #3715.

Sort-of-related: anyone interested in a ucx_info man page?

**Comment by shamisp** [comment]
@tonycurtis yes - we need man pages for EVERYTHING

**Comment by tonycurtis** [comment]
I have almost-ready templates for ucx_{info,perftest,read_profile}.1

**Comment by shamisp** [comment]
@tonycurtis - got for it.

Besides this, it would be useful to find a way to generate man pages for UCP.H. Doxygen can do it, but whatever it generates is not very pretty. Not sure what is the best way to resolve this.

**Comment by tonycurtis** [comment]
Would it be OK to do a PR to get the files in there, and then fix up the text later?  Someone else closer to the code might make a better description.

**Comment by tonycurtis** [comment]
Let's give it a go anyway, PR incoming...

**Comment by shamisp** [comment]
> Would it be OK to do a PR to get the files in there, and then fix up the text later? Someone else closer to the code might make a better description.

Yes, go for it.

**Comment by yosefe** [comment]
i think it's too late for v1.5, but we can still fix for v1.6.x (if relevant)

**Comment by yosefe** [comment]
@tonycurtis regarding the original reported issues - are you sure you are running the same ucx_info which you have installed from source build?
i've tried ucx-v1.5.2 release and don't see an issue:
```
$ tar xzf ucx-1.5.2.tar.gz && cd ucx-1.5.2
$ ./configure --prefix=$PWD/install --enable-mt
...

$ make -j10 install
...

$ ./src/tools/info/ucx_info -b|grep ENABLE_MT
#define ENABLE_MT                 1

$ ./install/bin/ucx_info -b|grep MT 
#define ENABLE_MT                 1

$ ./install/bin/ucx_info -v        
# UCT version=1.5.2 revision d4f8154
# configured with: --prefix=/tmp/ucx-1.5.2/install --enable-mt
```

**Comment by tonycurtis** [comment]
@yosefe does it work for you with an out-of-source-tree build?

That's the only thing I can see making a difference here.

**Comment by yosefe** [comment]
Looks like we have a problem creating dist tarballs by 'make dist', they already contain this file enen though  it should be generated only inside the build directory. checking ...

**Comment by yosefe** [comment]
fixed by @amaslenn in 6c34a554c69ca9486e790610a5761e1aded44bdb, for v1.6.x

**Comment by yosefe** [comment]
@tonycurtis can we close since it's fixed for v1.6.x?

**Comment by tonycurtis** [comment]
@yosefe  sounds like a plan!
